docs/Reference/CLI/CLI-Subcommands.md

```diff
@@ -310,14 +321,19 @@ The command accepts the following command line options:
 * [\--rpc-http-port](./CLI-Syntax.md#rpc-http-port)
 * [\--logging](./CLI-Syntax.md#logging)
 
+## `validate-config`
+
 === "Syntax"
 
     ```bash
-    besu retesteth [--data-path=<PATH>] [--rpc-http-host=<HOST>] [--rpc-http-port=<PORT>] [-l=<LOG VERBOSITY LEVEL>] [--host-allowlist=<hostname>[,<hostname>…]… or * or all]
+    besu validate-config --config-file <PATH-TO-CONFIG-FILE>
     ```
 
 === "Example"
 
     ```bash
-    besu retesteth --data-path=/home/me/me_node --rpc-http-port=8590 --host-allowlist=*
+    besu validate-config --config-file ../besu-local-nodes/config/besu/besu1.conf
     ```
+
+Performs basic syntax validation of the specified
```

Comment:
do we want to be explicit about what we mean by "basic syntax validation"

Reply:
basically
TOML syntax will be checked (is it valid format, checks for unmatched quotes etc) and unknown options will be flagged.
data types are NOT checked, and 
dependencies between options are not checked - this is only done at Besu startup